src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,46 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> data = readFrom(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        for (String line : data) {
+            if (line.contains(SUPPLY)) {
+                supply += Integer.parseInt(line.split(",")[1]);
+            } else if (line.contains(BUY)) {
+                buy += Integer.parseInt(line.split(",")[1]);
+            }
```

Comment:
if there is no third option, I leave just else without additional condition

Reply:
I've added it in case there will be more than two words. Refactored.